src/Illuminate/Support/Stringable.php

```diff
@@ -194,6 +194,25 @@ public function explode($delimiter, $limit = PHP_INT_MAX)
         return collect(explode($delimiter, $this->value, $limit));
     }
 
+    /**
+     * Split string by a regular expression.
+     *
+     * @param  string  $pattern
+     * @param  int  $limit
+     * @param  int  $flags
+     * @return \Illuminate\Support\Collection
+     */
+    public function split($pattern, $limit = -1, $flags = 0)
+    {
+        $keywords = preg_split($pattern, $this->value, $limit, $flags);
```

Comment:
What about error handling?

Reply:
`preg_split` returns `false` on failing

Reply:
Yes, and then you hide the error? Fail doesn't mean no matches, it means that the pattern was invalid, but you hide this.

Reply:
Ok get it! What's the recommended handling in this case? Removing the condition? Errors seem to be ignored for most methods (eg: https://github.com/laravel/framework/blob/87bb1352e9e15c28700ffdd3656b429ceddf4e7d/src/Illuminate/Support/Stringable.php#L336)